src/components/header/header.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+import classnames from 'classnames';
+
+import { Colors } from '../../constants';
+import { Styles } from './header.style';
+
+export const Header: FC = () => {
+  const classes = Styles(Colors);
+
+  return (
+    <header className={classes.header}>
+      <div className={classnames(classes.colorPanel, classes.headerEl)}>
+        {Object.values(Colors).map((color) => (
+          <div className={Styles({ color }).color} />
```

Comment:
```suggestion
          <div className={Colors({ color }).color} />
```
Maybe you`ll come up with another name